assets/js/components/event/MobileBottomPanel.tsx

```diff
@@ -0,0 +1,63 @@
+import { Channel } from "phoenix";
+import ProductsList from "./ProductsList";
+import ChatBox from "./ChatBox";
+import { ScreenTypeContext } from "../../utils/ScreenTypeContext";
+import { useContext } from "react";
+import type { ChatMessage, Client, Product } from "../../types/types";
+import "../../../css/event/mobilebottompanel.css";
+
+type Props = {
+  card: string;
+  client: Client;
+  eventChannel: Channel | undefined;
+  onBarClick: () => void;
+  products: Product[];
+  chatMessages: ChatMessage[];
+  isChatLoaded: boolean;
+  isBannedFromChat: boolean;
+};
+
+const selectAnimation = (card: string) => {
+  if (card === "hidden") return "";
+  if (card === "hide") return "SlideOutAnimation";
+  return "SlideInAnimation";
+};
+
+export const MobileBottomPanel = ({
+  onBarClick,
+  card,
+  client,
+  products,
+  chatMessages,
+  eventChannel,
+  isChatLoaded,
+  isBannedFromChat,
+}: Props) => {
+  const { device } = useContext(ScreenTypeContext);
+  const animation = selectAnimation(card);
+
+  if (device !== "mobile") return null;
+  return (
+    <div className={`MobileBottomPanel ${animation}`}>
+      <div className="MobileBottomPanel--TopBar">
+        <div className="MobileBottomPanel--TopBar--Bar" onClick={onBarClick} />
```

Comment:
As I don't know much about styling, I have a question whether this `--` in the name stands for something or it is simply convention?

Reply:
I removed the `--`